lib/TabTracker.js

```diff
@@ -81,10 +88,22 @@ TabTracker.prototype = {
     }
   },
 
-  logDeactivate: function(tab) {
-    if (this.isActivityStreamsURL(tab.url)) {
-      this.navigateAwayFromPage(tab);
-    }
+  logDeactivate: function(reason) {
+    return function(tab) {
+      if (this.isActivityStreamsURL(tab.url)) {
+        this.navigateAwayFromPage(tab, reason);
+      }
+    }.bind(this);
+  },
+
+  logClose: function(reason) {
+    return function(tab) {
+      if (tabs.activeTab.url != tab.url) {
```

Comment:
Via Travis-CI lint warnings:
```
/home/travis/build/mozilla/activity-streams/lib/TabTracker.js
  101:30  warning  Expected '!==' and instead saw '!='  eqeqeq
```

Not sure if we should promote that from a lint warning to a failure.